src/main/java/com/minecolonies/coremod/tileentities/TileEntityMultiBlock.java

```diff
@@ -0,0 +1,398 @@
+package com.minecolonies.coremod.tileentities;
+
+import com.google.common.primitives.Ints;
+import com.minecolonies.api.util.BlockPosUtil;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.api.util.BlockPosUtil'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)